lib/main_common.pm

```diff
@@ -1281,6 +1281,7 @@ sub load_x11tests {
             loadtest(is_sle('<15') ? "x11/rhythmbox" : "x11/gnome_music");
             loadtest "x11/wireshark";
             loadtest "x11/ImageMagick";
+            loadtest "x11/graphics_magick";
```

Comment:
Hey Ivan, since you only want to schedule your test for tumbleweed for now, I would suggest adding the `is_tumbleweed` condition, otherwise this will run also on SLE.

Reply:
It's good to exclude SLE but Leap also has GraphicsMagick so I would only add a `… unless is_sle;`

Reply:
however the whole block is only running for "is_sle" so this won't work

btw, as ImageMagick has the test module in CamelCase, care to rename graphics_magick in the same style?

Reply:
Thanks for your suggestions. I fixed it with a little refactoring of this condition statement. Please tell me if you agree with that